packages/components/addon/components/hds/form/field/index.hbs

```diff
@@ -0,0 +1,57 @@
+<div class={{this.classNames}}>
```

Comment:
Maybe there is a simpler approach:

**hds/form/field/index.hbs**
```
<div class={{this.classNames}}>
  {{yield (hash Label=(component "hds/form/label" controlId=this.id _contextualClass="hds-form-field__label"))}}
  {{yield
    (hash
      HelperText=(component "hds/form/helper-text" controlId=this.id _contextualClass="hds-form-field__helper-text")
    )
  }}
  {{yield (hash Control=(component "hds/yield"))}}
  {{yield (hash Error=(component "hds/form/error" controlId=this.id _contextualClass="hds-form-field__error"))}}
</div>
```

**hds/form/text-input/field.hbs** (for example)
```
<Hds::Form::Field as |F|>
  {{yield (hash Label=F.Label HelperText=F.HelperText Error=F.Error)}}
  <F.Control>
    <Hds::Form::TextInput::Base
      @type={{@type}}
      @value={{@value}}
      @isInvalid={{@isInvalid}}
      ...attributes
      id={{this.id}}
      aria-describedby={{this.ariaDescribedBy}}
    />
  </F.Control>
</Hds::Form::Field>
```

Let's discuss the pros/cons in a meeting.

Reply:
@alex-ju let me know if you want to rubber duck the problem